packages/export/package.json

```diff
@@ -34,5 +34,8 @@
     "lodash": "^4.17.21",
     "minimal-csv-formatter": "^1.0.15"
   },
-  "packageManager": "yarn@3.3.1"
+  "packageManager": "yarn@3.3.1",
+  "peerDependencies": {
+    "contexture-client": "workspace:^"
```

Comment:
I don't think we should be shipping with `workspace` in our dependencies 🤔

Reply:
Having workspace allows you to use the local version when developing instead of pulling a package from NPM.
We could do it for all the cross-dependencies.
Supposedly changeset should replace workspace with correct version on release
https://github.com/changesets/changesets/issues/432